helm-chart/renku-notebooks/values.yaml

```diff
@@ -75,14 +81,35 @@ tolerations: []
 
 affinity: {}
 
+serverOptions:
+  resources:
+    cpu_limit:
+      displayName: Number of CPUs
+      type: float
+      default: 1.0
+      range: [0.1, 4.0]
```

Comment:
only use discrete options

Reply:
why? CPUs requests are fractional

Reply:
a. are you going to use it?
b. with `options: [0, 1, 2, 4, 8]` you can have `0 = no limits`
c. you can include `0.5` anyway

Reply:
this is a limit, not a request -- so I think actually integer values are probably fine, so I propose we do `range: [1,X]` (I just pushed this change)